op-service/predeploys/addresses.go

```diff
@@ -102,6 +104,13 @@ func init() {
 			return config.GovernanceEnabled()
 		},
 	}
+	Predeploys["GovernanceDelegation"] = &Predeploy{
+		Address:       GovernanceDelegationAddr,
+		ProxyDisabled: true,
```

Comment:
This should have a proxy

Reply:
updated in https://github.com/ethereum-optimism/optimism/pull/11167/commits/85a0236c1aff5a2c5d9a30afc355b39a2c9ba3c3